source/opt/inline_pass.cpp

```diff
@@ -33,6 +33,14 @@ namespace opt {
 
 uint32_t InlinePass::FindPointerToType(uint32_t type_id,
                                        SpvStorageClass storage_class) {
+  opt::analysis::Type* pointeeTy = context()->get_type_mgr()->GetType(type_id);
```

Comment:
just analysis::Type.  It happens in other spots too.

Reply:
Will change